src/util/classes.ts

```diff
@@ -28,12 +28,17 @@ export const defaultCategories: Category[] = [
   },
   {
     name: ['Work', 'Programming'],
-    rule: { type: 'regex', regex: 'GitHub|Stack Overflow|BitBucket|Gitlab|vim|Spyder|kate' },
+    rule: {
+      type: 'regex',
+      regex: 'GitHub|Stack Overflow|BitBucket|Gitlab|vim|Spyder|kate|Ghidra|Scite',
+    },
   },
   {
     name: ['Work', 'Programming', 'ActivityWatch'],
     rule: { type: 'regex', regex: 'ActivityWatch|aw-', ignore_case: true },
   },
+  { name: ['Work', 'Image Editing'], rule: { type: 'regex', regex: 'Gimp|Inkscape' } },
+  { name: ['Work', 'Video Editing'], rule: { type: 'regex', regex: 'Kdenlive' } },
```

Comment:
```suggestion
  { name: ['Work', 'Image'], rule: { type: 'regex', regex: 'Gimp|Inkscape' } },
  { name: ['Work', 'Video'], rule: { type: 'regex', regex: 'Kdenlive' } },
  { name: ['Work', 'Audio'], rule: { type: 'regex', regex: 'Audacity' } },
  { name: ['Work', '3D'], rule: { type: 'regex', regex: 'Blender' } },
```

Removing the 'Editing' suffix to save space here. Not sure if this is the best call, but becomes repetitive otherwise (since the same suffix would be applied for Image/Video/Audio).